control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1218,6 +1221,232 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile default security group")
 	}
 
+	// TODO: consider using a side container in the etcd pod instead.
+	r.Log.Info("Reconciling etcd-backup cronJob")
+	if hostedControlPlane.Spec.Platform.Type == hyperv1.AWSPlatform {
+		configMapName := "etcd-backup-config" // TODO: get configMap name from annotation?
+		configMap := &corev1.ConfigMap{}
+		if err := r.Client.Get(ctx, client.ObjectKey{Namespace: hostedControlPlane.Namespace, Name: configMapName}, configMap); err != nil {
+			if !apierrors.IsNotFound(err) {
+				return fmt.Errorf("failed to get etcd backup configMap: %w", err)
+			}
```

Comment:
If the configmap is not found, we should delete the cronjob and svcacct if they exist. 
(Use support/util.DeleteIfNeeded)